src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -391,7 +391,7 @@ private void experimentalInterpretation(final JavaSparkContext ctx,
                         final String contigName = AlignedAssemblyOrExcuse.formatContigName(alignedAssembly.getAssemblyId(), contigIdx);
                         final List<AlignmentInterval> arOfAContig
                                 = getAlignmentsForOneContig(contigName, contigSequence, allAlignments.get(contigIdx), refNames, header);
-                        return new AlignedContig(contigName, contigSequence, arOfAContig, false);
+                        return new AlignedContig(contigName, contigSequence, arOfAContig);
```

Comment:
Can we change this variable name? `arOfAContig` is not obvious any more. Maybe `alignmentsForContig`?

Reply:
done